Optional `clipboard` for Gitflow commands

LuckyType opened the pull request:
I really like the Hotkey shortcuts to create feature/hotfix/bugfix branches, but i think using the clipboard is rather inconvenient.
This is a really amature idea for reading the user input to set the branch name including a slugify method.

There could be a setting to enable either clipboard / read input and another setting to enable automaticly slugifying given names.
I tried to add those settings myself within the .gitnow config file but didn't really understand how the parsing works and have given up.

Commit:
read user input for branch name

joseluisq commented:
Agree, honestly the default clipboard feature doesn't convince me as a default.
So making it as an optional entry on the config file should be the way to go.

In order to do that you need to read the new line of chars that you want to introduce in the config file first.

If you don't understand that then I could try to help some of those days.

joseluisq committed:
feat: optional clipboard for gitflow keybindings

it deactivates and makes optional the clipboard functionality when
gitflow keybindings are used and instead it reads text from the
standard input by default.

it introduces a new "options" section in order to control the
"clipboard" and useful for future new additions.

example:

[ options ]

clipboard = false

joseluisq commented:
@LuckyType I have resolved your issue with the parser. You can have a look at 99b7a89 to see the changes involved.
So feel free to give a test. I also tested it so after your check we could merge this.

joseluisq commented on conf.d/.gitnow:
```diff
@@ -1,3 +1,9 @@
+# Read text from system clipboard or from the standard input
+# only when using Gitflow keybindings
+clipboard = false
+
```
This is the new addition to test.

joseluisq commented:
Also it's not necessary to use or duplicate `__gitnow_slugify` since it's already used by every Gitflow command internally.
[__gitnow_gitflow_branch](https://github.com/joseluisq/gitnow/blob/master/functions/__gitnow_functions.fish#L109) does it.

joseluisq commented:
Released on [2.10.0](https://github.com/joseluisq/gitnow/releases/tag/2.10.0)
